Gemfile

```diff
@@ -11,10 +13,9 @@ gem "jquery-rails"
 # devise and activeadmin are used by the dummy application
 gem 'devise'
 gem 'activeadmin'
-gem "meta_search", '>= 1.1.0.pre'  # for activeadmin
 
 group :assets do
-  gem 'sass-rails',   '~> 3.2.3'
+  gem 'sass-rails'
```

Comment:
This gem is not necessary and should be removed.  See the following [comment](https://github.com/bespokepost/questionable/pull/22/files#r256635811).